README.md

```diff
@@ -46,21 +46,48 @@ Features include:
 Installing
 ==========
 
-There will _soon_ be a release of Swarm on Hackage and in a binary form!
-If you are interested in Haskell or functional programming, you can install
-it from source (see below) or, once it is released, with the Cabal tool.
-For instructions on how to install the binary directly, check the releases.
+**NOTE**: Swarm requires a POSIX-style terminal environment that
+supports `terminfo`.  Linux and MacOS should work fine.  On Windows,
+you will need to use something like the Windows Subsystem for Linux.
+For example, see [the comments on this GitHub
+issue](https://github.com/swarm-game/swarm/issues/53) for instructions
+on building from source on Windows.
 
-Installing from Source
-----------------------
+It is recommended that you use a relatively large terminal window
+(*e.g.* 170 columns x 40 rows or larger).  To find out the size of
+your terminal, you can type `stty size`. If it's not big enough, try
+decreasing the font size. You can read about and/or share recommended
+terminal settings in [this GitHub
+issue](https://github.com/swarm-game/swarm/issues/447).
 
-**NOTE**: Swarm requires a POSIX-style environment that supports
-`terminfo`.  On Linux and OSX, the below instructions should work
-natively.  On Windows, see [the comments on this GitHub
-issue](https://github.com/swarm-game/swarm/issues/53) for instructions
-getting it to work under the Windows Subsystem for Linux.
+- [Installing via binaries](#installing-via-binaries)
+- [Installing from Hackage](#installing-from-Hackage)
+- [Installing from source](#installing-from-source)
```

Comment:
GitHub already generates its TOC in its UI so this may be redundant, but I don't mind it. :slightly_smiling_face:

Reply:
Really? Where? I don't see any TOC...

Reply:
|![image](https://user-images.githubusercontent.com/44544735/194342141-7e98dab3-4fa9-46cb-86d3-be320aaae991.png)|
|-|